apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/config/PropertyBasedFileConfig.java

```diff
@@ -11,22 +11,23 @@
 
 import java.nio.file.Path;
 import javax.enterprise.inject.Produces;
+import javax.enterprise.inject.spi.CDI;
 import javax.inject.Inject;
 import javax.inject.Named;
 
 public class PropertyBasedFileConfig {
     private static final Logger LOG = LoggerFactory.getLogger(PropertyBasedFileConfig.class);
     private final PropertiesHolder propertiesHolder;
-
+    private static final AplCoreRuntime aplCoreRuntime = CDI.current().select(AplCoreRuntime.class).get();
```

Comment:
Make it non-static, (required for stateless multiple weld container init in unit tests )